script/scp-hosts.sh

```diff
@@ -40,8 +40,6 @@ do
   fi
 
   echo "scp dirs to $host/$installPath starting"
-	ssh -p $sshPort $host  "cd $installPath/; rm -rf bin/ master-server/ worker-server/ alert-server/ api-server/ ui/ tools/"
```

Comment:
Notice, I remove the `rm -rf ` command because it is a dangerous command, IMO, I rather have duplicated files than delete some wrong files